amazon-kinesis-client/src/main/java/software/amazon/kinesis/coordinator/logging/RejectedTaskEvent.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates.
+ * Licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package software.amazon.kinesis.coordinator.logging;
+
+import lombok.Getter;
+import lombok.ToString;
+
+@Getter
+@ToString
+public class RejectedTaskEvent extends ExecutorStateEvent {
```

Comment:
Where is this class being used? How is it being set within the KCL?

Reply:
Also would recommend naming classes based on what they are doing. `RejectedTaskEvent` mentions nothing on what the class is doing.

Reply:
This class will be used by an error handler in the Scheduler to react to rejected tasks, but that feature is currently being worked on by @ashwing. We will regroup to consolidate work

Reply:
All calls to these classes are now contained within this PR